apps/nrm/behaviours/generate-send-pdf.js

```diff
@@ -39,16 +39,34 @@ const deleteFile = (file) => {
   });
 };
 
+/**
+ * Get the user email from the session
+ *
+ * @param {*} req - request object
+ *
+ * @returns {string} - user's email
+ */
+const getUserEmail = (req) => {
+  return req.sessionModel.get('user-email');
+};
+
 module.exports = superclass => class extends superclass {
   async saveValues(req, res, next) {
     const tempName = createTemporaryFileName();
 
     const session = req.sessionModel.attributes;
     const data = await util.transformData(session);
     const file = await pdfPuppeteer.generate(templateFile, tempLocation, tempName, data);
+    const firstResponderEmail = getUserEmail(req);
```

Comment:
Do you need that function? Just get it I’d say

Functions are used generally when you are reusing or to simplify